Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-members = ["packages/*", "contracts/*", "tools/deploy"]
+members = ["packages/*", "contracts/*", "tools/deploy", "tools/verifier-helper"]
```

Comment:
Please move verifiers-helper to packages.

tools is for binaries not meant to be imported.
packages is for library code
contracts is for CosmWasm contracts
wasi is for AVS WASI code

(P.S. There is probably a merge conflict with the PR I just merged that updates this line, just use that one)

Reply:
thank you for elaborating, changed in b210cc859bc9